canonical/canonical-design
# WD-16705 Set up base template and flask app

## Description

- Initiated flask base app
- Created base template file and fixed index file
- Set up Vanilla styling
- Converted hack.css to styles.scss
- Build script for css
- cleaned up the file structure
- Transfered the old pages to `old-templates` folder

## QA
- Follow the steps in the readme to set up env and run the site.
- Go through the files to see all is good and correctly initiated.

## Fixes

- [WD-16705](https://warthogs.atlassian.net/browse/WD-16706)
- [WD- 16706](https://warthogs.atlassian.net/browse/WD-16706)

[WD-16705]: https://warthogs.atlassian.net/browse/WD-16705?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
